_includes/header.html

```diff
@@ -25,7 +25,7 @@
           <span class="status">Online</span>
         </div>
         <div class="apply">
-          <a href="" target="_blank">Get Passes <span><img src="/assets/images/forward.svg" /></span></a>
+          <a href="" target="_blank">Submit Proposal <span><img src="/assets/images/forward.svg" /></span></a>
```

Comment:
Please put `href="https://in.pycon.org/cfp/2020/proposals/"`

Also add `rel=nofollow noreferrer` to avoid the target="_blank" [vulnerabilities](https://dev.to/ben/the-targetblank-vulnerability-by-example)

Reply:
Didn't know about the target="_blank" vulnerability. Thank you for notifying!